src/postgres/utils/sql.ts

```diff
@@ -161,8 +161,7 @@ namespace sql {
   function escapeSqlIdentifier (str: string): string {
     let escaped = '"'
 
-    for (let i = 0; i < str.length; i++) {
-      let c = str[i]
+    for (const c of str) {
       if (c === '"') escaped += c + c
       else escaped += c
     }
```

Comment:
Seems there's not a huge amount in it, performance wise. I'd stick with the trusted `pg` implementation 👍 

https://jsperf.com/string-loop-scan-vs-replace